lips/lip-1.mdx

```diff
@@ -493,27 +425,188 @@ Represents a national ID.
 ```
 
 ### StatusEnum
-Valid values are:
+
+Valid values are the unicode strings:
 * `none` - No status is yet set from this actor.
 * `needs_kyc_data` - KYC data about the subaddresses is required by this actor.
-* `needs_recipient_signature` - Can only be associated with the sender actor.  Means that the sender still requires that the recipient VASP provide the signature so that the transaction can be put on-chain.
 * `ready_for_settlement` - Transaction is ready for settlement according to this actor (i.e. the required signatures/KYC data have been provided)
```

Comment:
Do we still want this state?  Looking at the diagram below, not sure it actually means anything anymore